src/data/markdown/translated-guides/en/05 Test Types/06-spike-testing.md

```diff
@@ -0,0 +1,96 @@
+---
+title: 'Spike testing'
+excerpt: 'Spike tests simulate moments of short, extreme load'
+---
+
+A spike test verifies whether the system survives and performs under sudden and massive rushes of utilization. It is one of the least common tests.
+
+Spike tests are useful when the system may experience events of sudden and massive traffic.
+Examples of such events include ticket sales (ComicCon, Taylor Swift), product launches (PS5, fashion apparel), sale announcements (Super Bowl ads), process deadlines (tax declaration), seasonal sales (Black Friday, Christmas, St.Valentine), etc.
+
+Spike testing increases to extremely high loads in a very short or non-existent ramp-up time.
+Usually, it has no plateau period or is very brief, as real users generally do not stick around doing extra steps in these situations. In the same way, the ramp-down is very fast or non-existent, letting the process iterate only once.
+
+This test usually includes very different processes than the previous test types, as these processes often aren't part of an average day in production. A recommendation is to automate the processes with heavy demand during the event. It may also require adding, removing, or modifying processes on the script that were not in the Average-Load tests.
```

Comment:
```suggestion
This test usually includes very different processes than the previous test types, as these processes often aren't part of an average day in production. A recommendation is to automate the processes with heavy demand during the event. It may also require adding, removing, or modifying processes on the script that were not in the average-load tests.
```